scss/mixins/_accessibility.scss

```diff
@@ -0,0 +1,14 @@
+//-----------------------------------
+//
+// Accessibility mixins
+// ====================
+//
+//-----------------------------------
+
+// Assert that the parent selector matches one of the selectors passed into $selectors
+@mixin assert-selector($selectors...) {
+  &:not(#{$selectors}) {
+    /*! You must ensure the element matches one of these selectors: #{$selectors} */
+    outline: 2px solid red !important;
```

Comment:
Maybe there's a way to wrap in a conditional `@if`. Can't think about it off-hand...sitting in a meeting :information_desk_person: